app/models/web_hook.rb

```diff
@@ -55,6 +55,10 @@ def self.enqueue_hooks(type, event, opts = {})
   end
 
   def self.enqueue_object_hooks(type, object, event, serializer = nil)
+    if type == :flag
+      Discourse.deprecate("The flags webhook is deprecated. Please use reviewable instead.")
```

Comment:
Wouldn't this end up spamming our logs in production since it is still being used? Also, I think it'll be helpful to include the version that we're removing this method in.

Reply:
I'm not sure how many people are actively using this particular web hook, but we should very quickly migrate them to the alternative. I would personally like to see the logs and take that responsibility on.

Reply:
I think deprecate is pretty safe it has callsite based protection so the flooding should not be severe.